shub/apps/library/views/helpers.py

```diff
@@ -189,9 +188,9 @@ def generate_collection_tags(collection):
     latest. Technically, collections should be namespaced
     consistently, and we assume this.
     """
-    unique_tags = set(
-        [c.tag for c in collection.containers.all() if not re.search(tag_regex, c.tag)]
-    )
+    unique_tags = {
```

Comment:
This needs to be a set to ensure we have uniqueness.

Reply:
Yes, `{ }` is the set constructor

Reply:
Totally did not realize that, my mind went to dict but you’re right, there aren’t key value pairs. Thanks for teaching me something new, this is ok then! :)